app/views/groups/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="group-index">
+  <nav class="navbar">
+ 
+    <div class="container-fluid">
+      <h2>Groups</h2>
+  </nav>
+  </div>
+  <% @groups.each do |group| %>
+    <%= link_to group_path(group) do %>
+    <div class="modal-body">
+      <div class='icons'><%= content_tag :i, "", class: "fa #{group.icon}" %></div>
+      <div><%= link_to "#{group.group_name}", group_path(group), class: 'no-underline' %></div>
+      <div><%= "$" "#{group.total_amount}" %></div>
+    </div>
```

Comment:
- [ Required ] I think it would be nice if you made sure that the styling for the categories page is consistent with the design on `Behance`. Currently, your design differs from the design on Behance in the following ways.
   
   - The padding around the list of items in your design is `small` compared to the padding on the Behance design which appears to be slightly `bigger`.

| Your design | Behance design |
|-----------------|------------------------|
| ![Screenshot from 2023-12-16 23-55-56](https://github.com/MayPyone/Budget-App/assets/16392046/6b5e48b2-0e51-4b61-97e0-e007ddcbcf4c) | ![Screenshot from 2023-12-16 23-54-20](https://github.com/MayPyone/Budget-App/assets/16392046/f355be9f-154c-4375-afcd-6214bc3725c7) |